### PR TITLE
Ease listing contributors

### PR DESCRIPTION
**Pull request by clementbiron**
Set up @all-contributors on this repository.

### CODE REVIEW

File: .all-contributors.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This file is required by [All Contributors](https://allcontributors.org/) to work properly and must not be deleted. Although it is automatically updated with the list of contributors, the actual output used in production is in JSON format and located in the `.all-contributorsrc` file.
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<!-- prettier-ignore-end -->
```
Comment (MattiSG) [inline .all-contributors.md:6]: > Shouldn't these be listed _before_ the start? I am concerned that these comments will be replaced by the AllContributors generated content.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (MattiSG) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
`contact@opentermsarchive.org`.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Contributing to this repository
+
+## Contributions to make
+
+You can add documents for tracking by submitting a pull request either using the GUI contributing tool, or by creating a JSON file. There are three kind of contributions you can make:
+
+- Adding declarations to a collection;
+- Updating declarations in a collection;
+- Removing declarations from a collection.
```
Comment (MattiSG) [inline CONTRIBUTING.md:9]: This doesn't seem appropriate for the `docs` repository 😅

**Review by MattiSG — CHANGES_REQUESTED** [review]
Seems like this work has been done too repeatedly 😉 this is not a collection!

File: .all-contributors.md (hunk 1 above)
Comment (clementbiron) [reply .all-contributors.md:6]: No, see the All Contributors [documentation](https://allcontributors.org/docs/fr/bot/installation).

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Contributing to this repository
+
+## 🚀 How to Contribute
+
+- You can check the open issues in this repository, assign yourself and solve them.
```
Comment (MattiSG) [inline CONTRIBUTING.md:5]: Unfortunately assigning has to be done by an admin, people can't self-assign.

```suggestion
- You can check the open issues in this repository, comment to state your interest and solve them.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- You can also make suggestions for additions or modifications of this documentation website.
```
Comment (MattiSG) [inline CONTRIBUTING.md:6]:
```suggestion
- You can also open issues to suggest additions or changes to this documentation website.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+## 📜 Read our Code of Conduct
+
+Before you begin coding and collaborating, please read our [Code of Conduct](./CODE_OF_CONDUCT.md) thoroughly to understand the standards (that you are required to adhere to) for community engagement. As part of our open-source community, we hold ourselves and other contributors to a high standard of communication. As a participant and contributor to this project, you are agreeing to abide by our Code of Conduct.
```
Comment (MattiSG) [inline CONTRIBUTING.md:10]:
```suggestion
Before you begin coding and collaborating, please read our [Code of Conduct](./CODE_OF_CONDUCT.md) to understand the standards (that you are required to adhere to) for community engagement. As part of our open-source community, we hold ourselves and other contributors to a high standard of communication. As a participant and contributor to this project, you are agreeing to abide by our Code of Conduct.
```

**Review by MattiSG — APPROVED** [review]

**Pull request merged**